src/cfn/template.yaml

```diff
@@ -1,105 +1,51 @@
-AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
This one looks like the autogenerated one in `templates/template.yaml`? I see some missing pieces (the vpc stuff), can you have a look?

Reply:
ah shit, that must have happened when i did that weird rebase stuff. i'll revert.

Reply:
*done

Reply:
hmmm, that didn't work

Reply:
That's better.